include/convex_bodies/vpolyintersectvpoly.h

```diff
@@ -322,6 +322,40 @@ class IntersectionOfVpoly {
     }
 
 
+    //------------------------------oracles for exponential sampling---------------//////
+
+    // compute intersection points of a ray starting from r and pointing to v
+    // with polytope discribed by A and b
+    std::pair<NT, int> quadratic_positive_intersect(Point const& r,
+                                    Point const& v,
+                                    VT const& Ac,
+                                    NT const& T,
+                                    VT& Ar,
+                                    VT& Av,
+                                    int& facet_prev) const
+    {
+        return std::make_pair(0, 0);
```

Comment:
Exceptions should be raised as in #142. The same applies for all classes where std::make_pair(0, 0) is called by actually no output is computed.

Reply:
@papachristoumarios Thank you for the comments!
This one changed according to https://github.com/GeomScale/volume_approximation/pull/142